index.html

```diff
@@ -17,122 +17,154 @@
     <header>
       <nav>
         <a href="#home"> <img src="images/My-Logo.png" /> </a>
-        <img src="images/Union.svg" />
+        <img class="hamburger" src="images/Union.svg" />
+        <div class="nav-list">
+          <span>Portfolio</span>
+          <span>About</span>
+          <span>Contact</span>
+        </div>
       </nav>
     </header>
     <div class="container">
-      <section>
-        <div id="text1">
-          <h1>I’m Eddy Glad to see you!</h1>
+      <section id="intro">
+        <div id="content">
+          <div id="text1">
+            <h1>I’m Eddy <br />Glad to see you!</h1>
+          </div>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me
+          </p>
+          <h4>LET’S CONNECT</h4>
```

Comment:
- [ ] I think it would be better if you add here more spacing to make your website identical to the Figma design as much as possible so kidnly add more margin/padding to it 🔥🚀📌

- Your design
![image](https://user-images.githubusercontent.com/58553711/200137668-f86680bd-f1a4-4f58-95a8-59497359b55c.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/200137674-835e0043-0e56-4448-9e19-4b6c826d21da.png)